server/src/main/java/org/eclipse/hono/server/MetricConstants.java

```diff
@@ -0,0 +1,32 @@
+package org.eclipse.hono.server;
+
+/**
+ * Constants/metric names
+ */
+public class MetricConstants {
+
+    public static String metricNameDownstreamConnections() {
+        return "hono.server.connections.downstream";
+    }
+
+    public static String metricNameDownstreamLinkCredits(String address) {
+        return "hono.server.link.downstream.credits." + address;
```

Comment:
I guess you're right, performance shouldn't be an issue since the compiler translates it to a `StringBuffer` anyways ...